src/Symfony/Contract/composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "symfony/contracts",
+    "type": "library",
+    "description": "A set of abstractions extracted out of the Symfony components",
+    "keywords": ["abstractions", "contracts", "decoupling", "interfaces", "interoperability", "standards"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3"
+    },
+    "require-dev": {
+        "cache/integration-tests": "dev-master"
```

Comment:
that's wrong. There is no such dev requirement

Reply:
Good catch, fixed thanks.